samples/basic/test/add.test.ts

```diff
@@ -1,17 +1,17 @@
-import { describe, expect, it } from 'vitest'
+import { expect } from 'chai'
```

Comment:
These are Vitest sample projects, so why changing this too?

Reply:
Ah yes. That's a brain fart moment and an oversight. Will correct it soon.